manifests/d/drakkan/SFTPGo/v2.2.1/drakkan.SFTPGo.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: drakkan.SFTPGo
+PackageVersion: v2.2.1
+Scope: machine
+InstallerSwitches:
+  Silent: "/VERYSILENT"
+  SilentWithProgress: "/SILENT"
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://github.com/drakkan/sftpgo/releases/download/v2.2.1/sftpgo_v2.2.1_windows_x86_64.exe
+  InstallerSha256: 9BA42FD43E3402165572D7A7C7F4E6CF10CEA927700F63EA714BA947DDE620F0
+- Architecture: x86
+  InstallerType: inno
+  Scope: machine
```

Comment:
```suggestion
```

Reply:
`Scope` is already defined as global.
`InstallerType` is being moved to be global.